Tuist/Config.swift

```diff
@@ -1,5 +1,5 @@
 import ProjectDescription
 
 let config = Config(
-    // cloud: .cloud(projectId: "tuist/tuist", url: "https://cloud.tuist.io")
+    cloud: .cloud(projectId: "tuist/tuist", url: "http://127.0.0.1:3000", options: [.optional])
```

Comment:
I guess this should point to `https://cloud.tuist.io` :D

Reply:
yes, of course 🤦 😄